datadog_checks_base/datadog_checks/base/checks/win/wmi/sampler.py

```diff
@@ -374,21 +374,21 @@ def build_where_clause(fltr):
             while f:
                 prop, value = f.popitem()
 
-                if isinstance(value, tuple):
+                if isinstance(value, (tuple, list)):
```

Comment:
Any situation when `value` will be a `tuple`?

Reply:
I think this would happen if we hardcoded a parsed instance (such as in a unit test) or had an instance that wasn't created from the parsed yaml. Including both types helps ensure cross-compatibility.

Reply:
I don't really see how you could get an instance with a tuple except when hardcoded in unit tests. If there's no reason to see a tuple here except of the way we wrote tests previously, tests should adapt to reflect the correct way.

Reply:
Might be something like https://github.com/DataDog/integrations-core/blob/master/active_directory/datadog_checks/active_directory/active_directory.py

Reply:
AD is using PDH though, but I dug through the code and we indeed need to keep tuples as well:
https://github.com/DataDog/integrations-core/blob/14ef6bd4ddcca25a901d7d78bf88a1f6a12e373a/win32_event_log/datadog_checks/win32_event_log/win32_event_log.py#L112